roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1187,11 +1189,12 @@ def get_defaults(self, roles):
                         iptables_sync_period='5s', set_node_ip=False)
             defaults['node'] = node
 
-        if 'nfs' in roles:
-            nfs = dict(exports_dir='/var/export', registry_volume='regvol',
-                       export_options='*(rw,sync,all_squash)')
-            defaults['nfs'] = nfs
-
+        defaults['hosted'] = dict(registry_storage_type=None,
+                                  registry_storage_host=None,
+                                  registry_storage_volume_dir='/var/exports',
+                                  registry_storage_volume='registry',
+                                  registry_storage_volume_capacity='5Gi',
+                                  registry_storage_export_options='*(rw,sync,all_squash)')
```

Comment:
Oh, and I'm not suggesting that we expose the dict out to the users in inventory (but we can and probably should do it as well, since it'll be easier to use in the installer wrapper config format), but rather how we use the internal representation of the variables after they have been generated by openshift_facts.